crates/turborepo-lockfiles/src/berry/mod.rs

```diff
@@ -756,7 +691,7 @@ mod test {
 
         assert_eq!(
             &lockfile.extensions,
-            &(["@babel/types@npm:^7.8.3", "lodash@npm:4.17.21"]
+            &(["@babel/types@npm:^7.8.3"]
```

Comment:
This test asserted some buggy behavior as it identified `lodash` as being an extension when it was really a user patch.

What we truly care about is that this `lodash` package gets included in the closure. This is asserted by the `test_closure_with_patch` test.